knox-homepage-ui/package.json

```diff
@@ -10,11 +10,13 @@
   },
   "private": true,
   "dependencies": {
+    "@angular/cdk": "^5.2.5",
```

Comment:
All the other angular versions is 5.2.0 but this is 5.2.5?

Reply:
I haven't tested with `5.2.0` as only `5.2.5` was listed as an officially supported version of Angular's material lib:

<img width="1672" alt="Screen Shot 2020-03-16 at 3 57 49 PM" src="https://user-images.githubusercontent.com/34065904/76771113-223b6400-679f-11ea-910f-d42a37fb4abc.png">